GameServerLib/Logic/GameObjects/AttackableUnits/AI/Champion.cs

```diff
@@ -33,6 +33,12 @@ public class Champion : ObjAIBase
         private uint _playerTeamSpecialId;
         private uint _playerHitId;
 
+        public bool canRecall = true;
+        private Buff visualBuff;
+        private Particle addParticle;
```

Comment:
`_addParticle`
Also, rename this. Sounds like this is a name of a function more than a variable.